src/knex-tables.d.ts

```diff
@@ -11,6 +11,36 @@ import { BreachDataTypes } from "./app/functions/universal/breach";
 
 // See https://knexjs.org/guide/#typescript
 declare module "knex/types/tables" {
+  interface AttributionRow {
+    id: number;
+    subscriber_id: number;
+    type: string; // first, last touch
```

Comment:
Noticed this `type` prop isn't in the `AttributionOptionalColumns` _or_ `AttributionAutoInsertedColumns` schemas below.

Reply:
it's neither optional nor auto inserted